app/controllers/job_runs_controller.rb

```diff
@@ -44,6 +43,26 @@ def download_report
     end
   end
 
+  def discovery_report_summary
+    @job_run = JobRun.find(params[:id])
+    if @job_run.report_ready?
+      @discovery_report = JSON.parse(File.read(@job_run.output_location))
+    else
+      flash.now[:warning] = 'There is no discovery report. Please check back later.'
+      render 'show'
+    end
+  end
+
+  def progress_log
+    @job_run = JobRun.find(params[:id])
+    if @job_run.progress_log_file_exists?
+      @progress_log = YAML.load_stream(File.read(@job_run.progress_log_file)) if @job_run.progress_log_file_exists?
+    else
+      flash.now[:warning] = 'Progress log file not available.'
+      render 'show'
+    end
+  end
+
```

Comment:
new views that show a discovery report summary and progress log file... we will use these in turbo-frames on the job run show page and load them async